### PR TITLE
Add code-blocks.md

### PR DESCRIPTION
**Pull request by fuchenshi**
### Description

Added an entry explaining the usage of code blocks in Markdown.

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@fuchenshi, I did a first review and this looks good as is. Ready for a second review.

File: content/markdown/concepts/code-blocks/code-blocks.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Code Blocks'
+Description: 'Code blocks can be used to hold multiple lines of code. They preserve formatting and provide syntax highlighting when possible.'
+Subjects:
+  - 'Developer Tools'
+  - 'Web Development'
+Tags:
+  - 'Markdown'
+  - 'GitHub'
```
Comment (caupolicandiaz) [inline content/markdown/concepts/code-blocks/code-blocks.md:9]:
```suggestion
  - 'GitHub'
  - 'Documentation' 
```

File: content/markdown/concepts/code-blocks/code-blocks.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+---
+Title: 'Code Blocks'
+Description: 'Code blocks can be used to hold multiple lines of code. They preserve formatting and provide syntax highlighting when possible.'
```
Comment (caupolicandiaz) [inline content/markdown/concepts/code-blocks/code-blocks.md:3]:
```suggestion
Description: 'Code blocks can be used to display multiple lines of code. They preserve formatting and provide syntax highlighting when possible.'
```

File: content/markdown/concepts/code-blocks/code-blocks.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  - 'Web Development'
+Tags:
+  - 'Markdown'
+  - 'GitHub'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+Code blocks can be used to hold multiple lines of code. They preserve formatting and provide syntax highlighting when possible.
```
Comment (caupolicandiaz) [inline content/markdown/concepts/code-blocks/code-blocks.md:15]:
```suggestion
Code blocks can be used to display multiple lines of code. They preserve formatting and provide syntax highlighting when possible.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@fuchenshi Just a few minor edits. Please tag me when you've updated the entry.

**Comment by fuchenshi** [comment]
@caupolicandiaz Thanks for the review! I've applied your suggestions to the file.

**Review by caupolicandiaz — APPROVED** [review]
@fuchenshi Looks good.

**Comment by yangc95** [comment]
Congrats on this entry @fuchenshi here's the live link: https://www.codecademy.com/resources/docs/markdown/code-blocks

Screenshot 2023-03-21 at 1 01 00 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
> 
> To add your GitHub username to your Codecademy profile:
> 
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
> 
> image
> 
> 2.) Then log out and log back into Codecademy using your GitHub login:
> 
> githublogin
>
> If the steps above do not work, it may be due to an email or local configuration issue.
>
> In addition, we may reach out in the future if any adjustments to these entries are required.

cc: @caupolicandiaz @SSwiniarski

**Pull request merged**